serf/serf.go

```diff
@@ -1098,6 +1102,10 @@ func (s *Serf) handleNodeLeaveIntent(leaveMsg *messageLeave) bool {
 		return false
 	}
 
+	// Always set the lamport time so that if we retransmit below it won't echo
+	// around forever!
+	member.statusLTime = leaveMsg.LTime
+
```

Comment:
Should we redo this in each case if we are going to be setting it outside?

Reply:
done